src/attribute.rs

```diff
@@ -286,6 +287,24 @@ impl Attribute {
         Ok(&self.value)
     }
 
+    pub fn to_ulong_array(&self) -> Result<Vec<CK_ULONG>> {
+        let ulen = std::mem::size_of::<CK_ULONG>();
```

Comment:
should we use here the `sizeof!()` macro? Given that it is already imported in this file, I think it would make it more consistent with the remaining code.

Reply:
sizeof!() is for places that need a CK_ULONG, this is a usize.